cob_teleop/ros/src/cob_teleop.cpp

```diff
@@ -343,14 +345,19 @@ void CobTeleop::updateBase(){
   }
 }
 
-
-void CobTeleop::joy_cb(const sensor_msgs::Joy::ConstPtr &joy_msg){
-
-  if(mode_switch_button_>=0 && mode_switch_button_<(int)joy_msg->buttons.size() && joy_msg->buttons[mode_switch_button_]==1)
+void CobTeleop::say(std::string text, bool blocking)
+{
+  cob_sound::SayGoal saygoal;
+  std::replace(text.begin(), text.end(), '_', ' ');
```

Comment:
addressed in https://github.com/ipa-fmw/cob_command_tools/commit/23cc62115a2c9df18d07c9454f833aff178f5d01